app/auth/README.md

```diff
@@ -0,0 +1,16 @@
+# Access Tokens
+All access tokens are valid for 30 minutes on the API. The endpoint /token generates the required access token to allow a user to access the service. As well as the /token endpoint, the oauth2passwordbearer form can also be used for authorization. When requesting information, the request needs to contain an authorization bearer token.
```

Comment:
I thought the `/token` endpoint used the OAuth2PasswordBearer form?

Should this read as:
"Access tokens are valid for 30 minutes. The `/token` endpoint generates the required access token to allow a user to access the service.  When submitting other requests, the request needs to contain this authorisation bearer token."